src/main.tsx

```diff
@@ -0,0 +1,23 @@
+/* @jsx createElement */
+import { createElement, render } from "./render";
+import { navigateTo, router } from "./routes";
+import { App } from ".";
+
+document.addEventListener("DOMContentLoaded", () => {
+  render(<App />, document.getElementById("root")!);
```

Comment:
```suggestion
  render(<App />, document.getElementById("root"));
```
Type Assertion은 타입 안정성을 저해시키기 때문에 가급적이면 사용을 피하는 것이 좋을 것 같습니다. 차라리 예외처리를 추가하는 것은 어떨까유??